app.js

```diff
@@ -28,6 +28,9 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 const index = require('./routes/index');
 app.use('/', index);
 
```

Comment:
On app.js you're not importing the session config, meaning that you will not be able to do the login if there is no session